components/header/styles.ts

```diff
@@ -21,14 +21,21 @@
 
 import { PrismTheme, createStyles } from "@solid/lit-prism-patterns";
 
+const HEADER_HEIGHT = "100px";
+
 const styles = (theme: PrismTheme) =>
   createStyles(theme, ["appLayout", "headerBanner"], {
     logoIndicatorContainer: {
       display: "flex",
+      height: HEADER_HEIGHT,
       flexDirection: "column",
       alignItems: "center",
+      justifyContent: "center",
       marginRight: theme.spacing(2),
-      paddingTop: theme.spacing(2),
+      paddingTop: theme.spacing(1),
```

Comment:
Why the need for paddingTop? This increases the total height to be over 100px (either 120px or 140px).

I would've hoped for a solution to be implemented in Prism? Any reason why you didn't prefer that?

Reply:
I just put a PR for another issue in Prism that might fix this so turning this into a draft